services/brig/test/integration/API/Provider.hs

```diff
@@ -599,8 +616,31 @@ testMessageBot config crt db brig galley cannon = withTestService config crt db
 
     testMessageBotUtil uid uc cid pid sid sref buf brig galley cannon
 
-testAddRemoveBotTeam :: Maybe Config -> FilePath -> DB.ClientState -> Brig -> Galley -> Cannon -> Http ()
-testAddRemoveBotTeam config crt db brig galley cannon = withTestService config crt db brig defServiceApp $ \sref buf -> do
+testBadFingerprint :: Config -> DB.ClientState -> Brig -> Galley -> Cannon -> Http ()
```

Comment:
Here's the test that checks for bad fingerprints, copypasted from `testMessageBot`